ledger-tool/src/output.rs

```diff
@@ -531,24 +641,23 @@ pub fn output_slot(
             for entry in entries.iter() {
                 num_hashes += entry.num_hashes;
             }
+            let blockhash = entries
+                .last()
+                .filter(|_| meta.is_full())
+                .map(|entry| entry.hash)
+                .unwrap_or(Hash::default());
```

Comment:
Note that our previous logic could misreport blockhash if we didn't have the full block; better to not report it (or report default hash) than to misreport it IMO